engine/src/main/java/org/terasology/logic/console/commands/ClientCommands.java

```diff
@@ -36,12 +37,21 @@
     private WorldProvider worldProvider;
 
 
+    /**
+     * Displays debug information on the target entity
```

Comment:
This is good but it just mimics what's in the description. Technically what is meant here is that debug information is displayed *for the target the camera is pointing at* which could help explain the command a little better. In other words it displays information for what the player is looking at.